typehints/esp8266/dht.pyi

```diff
@@ -0,0 +1,61 @@
+"""DHT driver
+
+Driver enables temperature and humidity readings from the DHT-11 and DHT-22 (also know as AM2302).  Humidity is
+measured as percentage of relative humidity and temperature is measured in Celsius.
+
+"""
+
+
+class DHTBase:
+    def __init__(self, pin: object) -> None:
```

Comment:
The type of `pin` can be narrowed to `machine.Pin` I guess. Are there any other types that the constructor accepts?

Reply:
Ok.  Does this require ```import machine``` to be added?